src/test/spec/v2_runner/mod.rs

```diff
@@ -39,9 +43,15 @@ const SKIPPED_OPERATIONS: &[&str] = &[
     "mapReduce",
 ];
 
-pub(crate) async fn run_v2_test(test_file: TestFile) {
+pub(crate) async fn run_v2_test(path: std::path::PathBuf, test_file: TestFile) {
     let internal_client = TestClient::new().await;
 
+    file_level_log(format!("Running tests from {}", path.display(),));
+    #[cfg(not(feature = "csfle"))]
+    let is_csfle_test = false;
+    #[cfg(feature = "csfle")]
+    let is_csfle_test = path.to_string_lossy().contains("client-side-encryption");
```

Comment:
why is the feature check needed here? shouldn't the check on the path always evaluate to false if the feature is off?

Reply:
Yup, you're right.  Fixed.